appveyor.yml

```diff
@@ -11,14 +11,14 @@ cache:
 # what combinations to test
 environment:
   matrix:
+    - nodejs_version: ''
```

Comment:
Is this equivalent to `stable`, which also resolves to 8 right now? Slightly clearer that way if so.